src/averageAges.js

```diff
@@ -1,3 +1,4 @@
+
 'use strict';
```

Comment:
Use strict is good but It'd be better to use Typescript to be even more strict :)